src/BudgetManager/BudgetManager.API/Seeding/SeedingService.cs

```diff
@@ -134,7 +142,12 @@ public async Task SeedUsers()
                 }
 
                 Console.WriteLine("Seeding Users...");
-                await _mediator.Send(new AddManyUsersCommand(users));
+                foreach (var user in users)
+                {
+                    var userId = await _mediator.Send(new AddUserCommand(user));
+                    await _authorizationManager.Register(user.Email, defaultPassword, userId);
+                }
+
```

Comment:
AddManyUsersCommand returns a list of Ids. So you can just do that and then do a 
for(int i = 0.........) {
var userId = userIds[i];
var user = users[i];
}
This would mean that you don't have to do so many calls to the database.